client/app/admin/create-project/create-project.html

```diff
@@ -57,63 +59,67 @@ <h1 class="section__title">
-                    <form class="form">
-                        <div class="form__group">
-                            <label class="form__option form__option--custom-checkbox">
-                                <input type="checkbox" name="form-custom-checkbox" id="form-custom-checkbox-1"
-                                       value="Checkbox 1"/>
-                                <span class="form__option__text">Roads</span>
-                                <span class="form__option__ui"></span>
-                            </label>
-                            <label class="form__option form__option--custom-checkbox">
-                                <input type="checkbox" name="form-custom-checkbox" value="form-custom-checkbox-2"/>
-                                <span class="form__option__text">Buildings</span>
-                                <span class="form__option__ui"></span>
-                            </label>
-                            <label class="form__option form__option--custom-checkbox">
-                                <input type="checkbox" name="form-custom-checkbox" value="form-custom-checkbox-3"/>
-                                <span class="form__option__text">Waterways</span>
-                                <span class="form__option__ui"></span>
-                            </label>
+                <div ng-show="createProjectCtrl.currentStep === 'taskSize'" class="content">
+                    <h2>Step 3: set task sizes</h2>
+                    <div>
+                        <p>Task size:</p>
+                        <div class="button-group button-group--horizontal" role="group">
+                            <button class="button button--achromic" type="button"
+                                    ng-click="createProjectCtrl.changeSizeTaskGrid(-1)">Larger
+                            </button>
+                            <button ng-class="createProjectCtrl.numberOfTasks > createProjectCtrl.maxNumberOfTasks ? 'disabled' : ''"
+                                    class="button button--achromic" type="button"
+                                    ng-disabled="createProjectCtrl.numberOfTasks > createProjectCtrl.maxNumberOfTasks"
+                                    ng-click="createProjectCtrl.changeSizeTaskGrid(1)">Smaller
+                            </button>
                         </div>
-                    </form>
+                        <hr/>
+                        <p>A new project will be created with {{ createProjectCtrl.numberOfTasks }} tasks.</p>
+                        <p>The size of each task is roughly {{ createProjectCtrl.sizeOfTasks | number : 2 }} km<sup>2</sup>.</p>
+                        <div ng-show="createProjectCtrl.numberOfTasks > createProjectCtrl.maxNumberOfTasks"
+                             class="alert alert--warning" role="alert">
+                          <p><strong>Warning:</strong> You can't create this project with smaller tasks.
```

Comment:
" Maximum number of tasks reached. If smaller tasks are required please make the area of interest smaller."